_data/navigation/en_US.yml

```diff
@@ -97,6 +100,9 @@ sidebar_pages:
   -
     title: Installing boot9strap (Soundhax)
     url: installing-boot9strap-(soundhax)
+  -
+    title: Installing boot9strap (Soundhax-sp)
```

Comment:
out of curiosity what is sp supposed to mean

Reply:
special